lib/smart_app_launch/standalone_launch_group_stu2.rb

```diff
@@ -39,5 +40,10 @@ class StandaloneLaunchGroupSTU2 < StandaloneLaunchGroup
         }
       }
     )
+
+    test from: :smart_app_redirect_stu2
+
+    redirect_index = children.find_index { |child| child.id.to_s.end_with? 'app_redirect' }
+    children[redirect_index] = children.pop
```

Comment:
This might not be super common, but it might be nice to have test replacement as part of the DSL.

e.g.

```ruby
test from: :smart_app_redirect_stu2, replaces: smart_app_redirect
```